contributing/development/core_and_modules/core_types.rst

```diff
@@ -103,7 +103,25 @@ which are equivalent to new, delete, new[] and delete[].
 memnew/memdelete also use a little C++ magic and notify Objects right
 after they are created, and right before they are deleted.
 
-For dynamic memory, use Vector<>.
+For dynamic memory, use one of Godot's sequence types such as ``Vector<>``
+or ``LocalVector<>``. ``Vector<>`` behaves much like an STL ``Vector<>``,
```

Comment:
```suggestion
or ``LocalVector<>``. ``Vector<>`` behaves much like an STL ``std::vector<>``,
```

Reply:
...and that definitely counts as something silly which slipped through. Thank you for the catch!